| No. | issue | title | labels | assignee | bot advice | created date |
| ------ | ------ | ------ | ------ | ------ | ------ | ------ |
| 1 | [#21665](https://github.com/Azure/azure-sdk-for-python/issues/21665) | Typo in parameters | ['bug', 'Authorization', 'customer-reported', 'needs-team-attention', 'CXP Attention'] | ['MughundhanRaveendran-MSFT'] | new comment ! | 11-09  |
| 2 | [#21664](https://github.com/Azure/azure-sdk-for-python/issues/21664) | Documentation fix: <br/>Expectedtype 'TokenCredential', <br/>got'UserPassCredentials' <br/>instead | ['Docs', 'Network - DNS', 'customer-reported', 'needs-team-attention', 'CXP Attention'] | ['PramodValavala-MSFT'] | new comment ! | 11-09  |
| 3 | [#21642](https://github.com/Azure/azure-sdk-for-python/issues/21642) | 'MsiCredential' object <br/>hasno attribute <br/>'signed_session' | ['customer-reported', 'Azure.Identity', 'Subscription', 'needs-team-attention'] | ['BigCat20196'] | new comment ! | 11-08  |
| 4 | [#21635](https://github.com/Azure/azure-sdk-for-python/issues/21635) | support to create/update <br/>VpnSiteLinksOperations | ['feature-request', 'Service Attention', 'customer-reported', 'needs-team-attention', 'Network - Private Link'] | ['RAY-316'] |   | 11-07  |
| 5 | [#21467](https://github.com/Azure/azure-sdk-for-python/issues/21467) | B2B invitation endpoint <br/>SDK | ['feature-request', 'Graph', 'Service Attention', 'customer-reported', 'needs-team-attention'] |   | new comment ! | 10-28  |
| 6 | [#21451](https://github.com/Azure/azure-sdk-for-python/issues/21451) | Azure function list <br/>properties | ['question', 'Service Attention', 'customer-reported', 'Web Apps', 'needs-team-attention'] | ['BigCat20196'] | new comment ! | 10-27  |
| 7 | [#21450](https://github.com/Azure/azure-sdk-for-python/issues/21450) | Network resource provider <br/>APIversionis 8 versions behind the <br/>lateststable version | ['bug', 'Network', 'Service Attention'] |   | new comment ! | 10-27  |
| 8 | [#21449](https://github.com/Azure/azure-sdk-for-python/issues/21449) | Authorization Role <br/>DefinitionUpdate The requested <br/>resourcedoes not support http <br/>method'PUT': | ['question', 'Authorization', 'Service Attention', 'customer-reported', 'needs-team-attention'] | ['SaurabhSharma-MSFT'] | new comment ! | 10-27  |
| 9 | [#21375](https://github.com/Azure/azure-sdk-for-python/issues/21375) | I can not update only one <br/>key=valuefor the web app <br/>applicationsetting | ['feature-request', 'Service Attention', 'customer-reported', 'needs-team-attention'] | ['SaurabhSharma-MSFT'] | new comment ! | 10-22  |
| 10 | [#21301](https://github.com/Azure/azure-sdk-for-python/issues/21301) | [Python] reduce size for <br/>`azure-mgmt-network` | [] | ['RAY-316'] |   | 10-18  |
| 11 | [#21252](https://github.com/Azure/azure-sdk-for-python/issues/21252) | from <br/>._application_gateways_operationsimport <br/>ApplicationGatewaysOperationsValueError: source code <br/>stringcannot contain null bytes | ['bug', 'Network - Application Gateway', 'Service Attention', 'customer-reported', 'needs-team-attention'] | ['RAY-316'] | new comment ! | 10-14  |
| 12 | [#21008](https://github.com/Azure/azure-sdk-for-python/issues/21008) | KeyInfo for functions <br/>doesn'tserialize correctly | ['bug', 'feature-request', 'Service Attention', 'customer-reported', 'Web Apps', 'needs-team-attention'] | ['BigCat20196'] |   | 10-01  |
| 13 | [#20907](https://github.com/Azure/azure-sdk-for-python/issues/20907) | Get the Azure VM IP <br/>information | ['question', 'Compute', 'customer-reported', 'needs-team-attention'] | ['RAY-316'] |   | 09-28  |
| 14 | [#20804](https://github.com/Azure/azure-sdk-for-python/issues/20804) | Switching Tenant without <br/>relogin | ['feature-request', 'Service Attention', 'customer-reported', 'ARM - Core', 'needs-team-attention'] |   | new comment ! | 09-23  |
| 15 | [#20802](https://github.com/Azure/azure-sdk-for-python/issues/20802) | Unable to authenticate <br/>AzureStackusing ADFS authentication <br/>withUserPassCredentials | ['bug', 'Azure Stack', 'Service Attention', 'customer-reported', 'needs-team-attention'] |   | new comment ! | 09-23  |
| 16 | [#20679](https://github.com/Azure/azure-sdk-for-python/issues/20679) | Query ALL azure services <br/>availablein EACH region by azure <br/>sdkfor python | ['question', 'customer-reported'] |   |   | 09-14  |
| 17 | [#20664](https://github.com/Azure/azure-sdk-for-python/issues/20664) | Azure data factory <br/>triggercreation using python SDK | ['question', 'Data Factory', 'Service Attention', 'customer-reported'] |   | new comment ! | 09-13  |
| 18 | [#20629](https://github.com/Azure/azure-sdk-for-python/issues/20629) | `top` parameter in <br/>ConsumptionManagementClientdidn't return expecting <br/>list. | ['question', 'Service Attention', 'customer-reported', 'needs-team-attention', 'Consumption - UsageDetailsAndExport'] | ['BigCat20196'] | new comment ! | 09-09  |
| 19 | [#20372](https://github.com/Azure/azure-sdk-for-python/issues/20372) | Azure python api calls <br/>getsstuck sometimes for a <br/>longtime | ['bug', 'Compute', 'Service Attention', 'customer-reported', 'needs-team-attention'] | ['jaylabell'] | new comment ! | 08-20  |
| 20 | [#20356](https://github.com/Azure/azure-sdk-for-python/issues/20356) | azure-mgmt-netapp: <br/>Operation:'Change Ownership' is not <br/>permittedwhen creating NFSv4.1 <br/>volume | ['bug', 'customer-reported', 'needs-team-attention'] | ['RAY-316'] |   | 08-19  |
| 21 | [#20288](https://github.com/Azure/azure-sdk-for-python/issues/20288) | The operation api.get in <br/>themodule <br/>azure.mgmt.apimanagement.operationsdoesn't return all the <br/>fields | ['bug', 'API Management', 'customer-reported', 'needs-team-attention'] | ['BigCat20196'] | new comment ! | 08-16  |
| 22 | [#20181](https://github.com/Azure/azure-sdk-for-python/issues/20181) | azure-mgmt-redis: LICENSE <br/>missingfrom PyPi | ['bug', 'customer-reported', 'needs-team-attention'] |   |   | 08-09  |
| 23 | [#20177](https://github.com/Azure/azure-sdk-for-python/issues/20177) | <br/>azure-mgmt-recoveryservices:LICENSE missing from PyPi | ['bug', 'customer-reported', 'needs-team-attention'] |   |   | 08-09  |
| 24 | [#20175](https://github.com/Azure/azure-sdk-for-python/issues/20175) | azure-mgmt-rdbms: LICENSE <br/>missingfrom PyPi | ['bug', 'customer-reported', 'needs-team-attention'] |   |   | 08-09  |
| 25 | [#20172](https://github.com/Azure/azure-sdk-for-python/issues/20172) | <br/>azure-mgmt-policyinsights:LICENSE missing from PyPi | ['bug', 'customer-reported', 'needs-team-attention'] |   |   | 08-09  |
| 26 | [#20170](https://github.com/Azure/azure-sdk-for-python/issues/20170) | azure-mgmt-maps: LICENSE <br/>missingfrom PyPi | ['bug', 'customer-reported'] |   |   | 08-09  |
| 27 | [#20168](https://github.com/Azure/azure-sdk-for-python/issues/20168) | azure-mgmt-loganalytics: <br/>LICENSEmissing in PyPi | ['bug', 'customer-reported', 'Monitor - LogAnalytics', 'needs-team-attention'] |   |   | 08-09  |
| 28 | [#20166](https://github.com/Azure/azure-sdk-for-python/issues/20166) | azure-mgmt-iotcentral: <br/>LICENSEmissing in PyPi | ['bug', 'IoT', 'customer-reported', 'needs-team-attention'] |   |   | 08-09  |
| 29 | [#20163](https://github.com/Azure/azure-sdk-for-python/issues/20163) | azure-mgmt-datamigration: <br/>LICENSEmissing in PyPi | ['bug', 'Data Migration', 'customer-reported', 'needs-team-attention'] |   |   | 08-09  |
| 30 | [#20161](https://github.com/Azure/azure-sdk-for-python/issues/20161) | azure-mgmt-databoxedge: <br/>LICENSEmissing in PyPi | ['bug', 'customer-reported', 'needs-team-attention', 'DataBox Edge'] |   |   | 08-09  |
| 31 | [#20159](https://github.com/Azure/azure-sdk-for-python/issues/20159) | azure-mgmt-compute: <br/>LICENSEmissing in PyPi | ['bug', 'Compute', 'customer-reported', 'needs-team-attention'] |   |   | 08-09  |
| 32 | [#20155](https://github.com/Azure/azure-sdk-for-python/issues/20155) | azure-mgmt-batchai: <br/>LICENSEfile missing in pypi | ['bug', 'BatchAI', 'customer-reported', 'needs-team-attention'] |   |   | 08-09  |
| 33 | [#20044](https://github.com/Azure/azure-sdk-for-python/issues/20044) | Follow-up: Is it possible <br/>tocreate a VM with OS disk <br/>typedifferent than Premium <br/>SSD? | ['question', 'Compute - VM', 'Service Attention', 'customer-reported', 'needs-team-attention'] | ['RAY-316', 'oriwolman'] | new comment ! | 08-02  |
| 34 | [#19793](https://github.com/Azure/azure-sdk-for-python/issues/19793) | [azure-mgmt-sql] add <br/>supportfor SQL replica | ['feature-request', 'SQL', 'Service Attention', 'customer-reported', 'needs-team-attention'] | ['RAY-316'] | new comment ! | 07-14  |
| 35 | [#19665](https://github.com/Azure/azure-sdk-for-python/issues/19665) | Error responses from <br/>certainoperations do not conform <br/>toODataV4 | ['question', 'App Services', 'Service Attention', 'customer-reported'] | ['BigCat20196', 'RAY-316'] |   | 07-06  |
| 36 | [#19663](https://github.com/Azure/azure-sdk-for-python/issues/19663) | azure-mgmt-web 3.0.0: <br/>breakingchanges without migration <br/>information | ['question', 'App Services', 'Service Attention', 'customer-reported'] | ['RAY-316'] | new comment ! | 07-06  |
| 37 | [#19600](https://github.com/Azure/azure-sdk-for-python/issues/19600) | consumption SDK <br/>usage_details.listdoes not filter by date | ['bug', 'Service Attention', 'customer-reported', 'Consumption - UsageDetailsAndExport'] |   | new comment ! | 06-30  |
| 38 | [#19585](https://github.com/Azure/azure-sdk-for-python/issues/19585) | Aggregation strings <br/>aren'tlisted | ['question', 'Monitor', 'Service Attention', 'customer-reported'] | ['RAY-316'] | new comment ! | 06-29  |
| 39 | [#19294](https://github.com/Azure/azure-sdk-for-python/issues/19294) | Which SDK to be used in <br/>orderto create a Express route <br/>circuitfrom a Direct Port? | ['question', 'Network', 'Service Attention', 'customer-reported'] |   | new comment ! | 06-17  |
| 40 | [#19293](https://github.com/Azure/azure-sdk-for-python/issues/19293) | Add possibility to get <br/>tagson subscription level | ['feature-request', 'Service Attention', 'customer-reported', 'Subscription'] |   | new comment ! | 06-17  |
| 41 | [#19224](https://github.com/Azure/azure-sdk-for-python/issues/19224) | azure-mgmt-batch Pool <br/>createincorrect API version | ['bug', 'Batch', 'Service Attention', 'customer-reported'] | ['RAY-316'] | new comment ! | 06-11  |
| 42 | [#19203](https://github.com/Azure/azure-sdk-for-python/issues/19203) | Cross references are <br/>broken | ['Docs', 'question', 'customer-reported', 'Subscription'] | ['RAY-316'] | new comment ! | 06-10  |
| 43 | [#19178](https://github.com/Azure/azure-sdk-for-python/issues/19178) | azure-mgmt-monitor <br/>throwingDeserializationError when <br/>handlingAPI error | ['question', 'Monitor', 'customer-reported'] |   | new comment ! | 06-09  |
| 44 | [#19147](https://github.com/Azure/azure-sdk-for-python/issues/19147) | <br/>backup_protection_containers.listgot 400 error | ['question', 'Recovery Services Backup', 'customer-reported'] | ['RAY-316'] | new comment ! | 06-08  |
| 45 | [#19009](https://github.com/Azure/azure-sdk-for-python/issues/19009) | azure-mgmt-resource <br/>documentationis missing the module <br/>itself | ['bug', 'Docs', 'customer-reported'] | ['RAY-316'] |   | 05-31  |
| 46 | [#18957](https://github.com/Azure/azure-sdk-for-python/issues/18957) | azure-mgmt-costmanagement <br/>CostManagementClientdoes not handle <br/>api_version | ['bug', 'Service Attention', 'Cost Management - Query'] |   | new comment ! | 05-26  |
| 47 | [#18937](https://github.com/Azure/azure-sdk-for-python/issues/18937) | Varying spelling of <br/>CBLMarinerand CBL_MARINER | ['question', 'Service Attention', 'Container Service'] |   | new comment ! | 05-25  |
| 48 | [#18928](https://github.com/Azure/azure-sdk-for-python/issues/18928) | Missing ImportSource (and <br/>others)in <br/>azure.mgmt.containerregistry.v2019_04_01.models | ['question', 'Container Registry', 'customer-reported'] |   | new comment ! | 05-25  |
| 49 | [#18927](https://github.com/Azure/azure-sdk-for-python/issues/18927) | azure-mgmt-costmanagement <br/>CostManagementClientignores nextLink for <br/>usage | ['bug', 'Service Attention', 'Cost Management - UsageDetailsAndExport'] | ['ShilpiGautam', 'olsaal-msft'] | new comment ! | 05-25  |
| 50 | [#18911](https://github.com/Azure/azure-sdk-for-python/issues/18911) | Network interface cannot <br/>bemodified by user Code: <br/>CannotModifyNicAttachedToPrivateEndpoint | ['bug', 'question', 'Network', 'Service Attention', 'customer-reported'] |   | new comment ! | 05-25  |
| 51 | [#18905](https://github.com/Azure/azure-sdk-for-python/issues/18905) | Bad body serialization <br/>creatinga Schedule Trigger in <br/>DataFactory | ['bug', 'Data Factory', 'Service Attention', 'customer-reported', 'needs-team-attention'] |   | new comment ! | 05-25  |
| 52 | [#18877](https://github.com/Azure/azure-sdk-for-python/issues/18877) | Python example invalid | ['bug', 'customer-reported'] |   |   | 05-24  |
| 53 | [#18873](https://github.com/Azure/azure-sdk-for-python/issues/18873) | The Python SDK passing <br/>parametersto adf pipeline does not <br/>work | ['bug', 'Data Factory', 'customer-reported', 'needs-team-triage'] | ['00Kai0'] | new comment ! | 05-24  |
| 54 | [#18809](https://github.com/Azure/azure-sdk-for-python/issues/18809) | hardcoded API version in <br/>azure/mgmt/sql/operations/_databases_operations.pydoes not work for all <br/>sovereignclouds | ['feature-request', 'SQL', 'Service Attention', 'customer-reported'] |   |   | 05-19  |
| 55 | [#18788](https://github.com/Azure/azure-sdk-for-python/issues/18788) | [API Management] Python <br/>DSKnot using all schema data | ['bug', 'API Management', 'customer-reported'] |   |   | 05-18  |
| 56 | [#18433](https://github.com/Azure/azure-sdk-for-python/issues/18433) | [proposal not a bug] <br/>missingexample how to proceed <br/>withB2CTenantsOperations <br/>class | ['feature-request', 'customer-reported', 'needs-team-triage'] |   |   | 04-29  |
| 57 | [#18102](https://github.com/Azure/azure-sdk-for-python/issues/18102) | Content too old to follow | ['bug', 'Docs', 'question', 'MySQL', 'PostgreSQL', 'Service Attention', 'customer-reported'] |   | new comment ! | 04-16  |
| 58 | [#17801](https://github.com/Azure/azure-sdk-for-python/issues/17801) | Azure SDK is over 500MB <br/>andgrowing on each release. | ['question', 'Network', 'Service Attention', 'customer-reported'] | ['lmazuel'] | new comment ! | 04-05  |
| 59 | [#17764](https://github.com/Azure/azure-sdk-for-python/issues/17764) | Implement <br/>ExternalBearerTokenCredentialPolicyin azure-mgmt-core | ['Azure.Mgmt.Core'] | ['jiasli', '00Kai0'] | new comment ! | 04-02  |
| 60 | [#17439](https://github.com/Azure/azure-sdk-for-python/issues/17439) | Category of the <br/>sub-assessment | ['question', 'Security', 'Service Attention', 'customer-reported'] |   | new comment ! | 03-18  |
| 61 | [#16943](https://github.com/Azure/azure-sdk-for-python/issues/16943) | Feature To Append Tags To <br/>AResource | ['feature-request', 'customer-reported', 'needs-team-attention', 'needs-team-triage'] | ['xiangyan99', '00Kai0', 'RAY-316', 'jaylabell'] |   | 02-25  |
| 62 | [#15925](https://github.com/Azure/azure-sdk-for-python/issues/15925) | get public IP address at <br/>thesame time when I get VM <br/>information. | ['question', 'Compute', 'Compute - VM', 'customer-reported'] | ['00Kai0'] | new comment ! | 12-28  |
| 63 | [#15913](https://github.com/Azure/azure-sdk-for-python/issues/15913) | AzureFileVolume not <br/>unmountedon container delete <br/>done_callback | ['question', 'customer-reported'] |   | new issue ! | 12-23  |
| 64 | [#15910](https://github.com/Azure/azure-sdk-for-python/issues/15910) | Creating <br/>ContainerInstanceManagementClientusing client factory <br/>causesTypeError exception | ['question', 'customer-reported'] |   | new comment ! | 12-23  |
| 65 | [#15857](https://github.com/Azure/azure-sdk-for-python/issues/15857) | Subscription scope usage <br/>isnot supported for current <br/>apiversion | ['question', 'customer-reported'] |   | new issue ! | 12-17  |
| 66 | [#15732](https://github.com/Azure/azure-sdk-for-python/issues/15732) | Cannot Regenerate Storage <br/>AccountAccess Key | ['Docs', 'question', 'customer-reported'] |   | new comment ! | 12-09  |
| 67 | [#15684](https://github.com/Azure/azure-sdk-for-python/issues/15684) | Get all 'Global <br/>administrators'users in Azure using <br/>python | ['question', 'Service Attention', 'customer-reported'] |   | new issue ! | 12-07  |
| 68 | [#15008](https://github.com/Azure/azure-sdk-for-python/issues/15008) | KeyVault library returns <br/>lessnumber of fields when <br/>comparedto PowerShell command | ['question', 'KeyVault', 'customer-reported', 'needs-team-triage'] |   | new comment ! | 11-04  |
| 69 | [#13943](https://github.com/Azure/azure-sdk-for-python/issues/13943) | Error validating/creating <br/>StorageAccount | ['question', 'Service Attention', 'customer-reported'] |   |   | 09-22  |
